pkg/types/position.go

```diff
@@ -218,14 +218,22 @@ func (p *Position) GetBase() (base fixedpoint.Value) {
 	return base
 }
 
-// GetBaseAndAverageCost locks the mutex and return the base quantity and average cost
-func (p *Position) GetBaseAndAverageCost() (base fixedpoint.Value, averageCost fixedpoint.Value) {
```

Comment:
you are actually changing an existing public API which could be used from somewhere else, it could be a private repository or a private strategy where we can't scan the reference

Reply:
we need to be careful when changing an existing public api